### PR TITLE
Style: Use short block notation for simple one-liners

### PR DESCRIPTION
**Pull request by Sija**
This PR changes simple calls on the single block receiver into their short form:

```cr
%w(foo bar).map { |v| v.upcase }
# into
%w(foo bar).map(&.upcase)
```

This IMHO makes code more succinct and crystal-ish :)

### CODE REVIEW

File: samples/havlak.cr [hunk 1]
```diff
@@ -183,7 +183,7 @@ class UnionFindNode
       node = parent
     end
 
-    node_list.each { |iter| iter.parent = node.parent }
+    node_list.each(&.parent=(node.parent))
```
Comment (yxhuvud) [inline samples/havlak.cr:186]: Most of these are great, but I'm not certain I like changing the assignments.

Comment (straight-shoota) [reply samples/havlak.cr:186]: I agree. Assignments with parentheses look weird.

Comment (Sija) [reply samples/havlak.cr]: I've force-pushed with assignments shorthands removed.

**Review by straight-shoota — APPROVED** [review]

**Comment by Sija** [comment]
Anything else left to do here?

**Review by beta-ziliani — APPROVED** [review]
Thanks @Sija !

**Pull request merged**